### PR TITLE
Allow css tag for everything (*).

### PR DESCRIPTION
**Pull request by Hamzantal**
This will help developers who need to style all tags.

### CODE REVIEW

**Comment by Hypnosphi** [comment]
I wonder if we can utilize `*` operator overloading for that

File: kotlin-css/src/main/kotlin/kotlinx/css/TagSelectors.kt [hunk 1]
```diff
@@ -2,6 +2,8 @@ package kotlinx.css
 
 class TagSelector(val tagName: String)
 
+val star get() = TagSelector("*")
```
Comment (Hypnosphi) [inline kotlin-css/src/main/kotlin/kotlinx/css/TagSelectors.kt]: It's not actually a tag

Comment (Hypnosphi) [reply kotlin-css/src/main/kotlin/kotlinx/css/TagSelectors.kt]: I think you should rather add following member function to https://github.com/JetBrains/kotlin-wrappers/blob/master/kotlin-css/src/main/kotlin/kotlinx/css/CSSBuilder.kt#L34

```
fun star(block: RuleSet) = "*"(block)
```

The "star" name is also disputable. I would prefer something more semantic, like "everything" or "wildcard"

Comment (Leonya) [reply kotlin-css/src/main/kotlin/kotlinx/css/TagSelectors.kt]: Jeez, what was I thinking?

**Comment by Hamzantal** [comment]
Oh my god. My bad. My apologies

**Pull request merged**